feat(curriculum): dice game project (#50819)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/64a1b863a27764030bc712fd.md

---
id: 64a1b863a27764030bc712fd
title: Step 27
challengeType: 0
dashedName: step-27
---

# --description--

> As you can see that the text on the rules button is changing but the rules are not being shown. To fix that, use `querySelector` to target the div with the class of `rules-container` and store it in a variable named `rulesContainer`.

Reviewer: ```suggestion
If you click on the `Show rules` button, you can see the text change, but the rules are not being shown.

Use `querySelector` to target the element with the class of `rules-container` and store it in a variable named `rulesContainer`.
```

> As you can see that the text on the rules button is changing but the rules are not being shown. For that, on the HTML there's a div with a class named `rules-container`. Target that using a `querySelector` and store it in a variable named `rulesContainer`.

Reviewer 2: ```suggestion
As you can see that the text on the rules button is changing but the rules are not being shown. To fix that, target the div with the class `rules-container` using a `querySelector` and store it in a variable named `rulesContainer`.
```
